script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -711,5 +711,6 @@ We'd really be in trouble if the sort of monster that can overturn a plan hundre
 [sha:780e3ac380b75b73ae94e1fce5ffb556e22840c8]{
 -- Page 619, Offset 18141.
 -- それこそ自衛隊の出番ではないだろうか。
-If they did, I guess we'd be better off calling in the Japanese Self-Defense Force.
+// pardon the nitpicking but is it really necessary to add the word japanese since we know the setting?
```

Comment:
yeah, this is fine. The Japanese word is 自衛隊 which has no reference (implicitly or otherwise) to Japan. For example, in the US we don't call it the US Army, we just say "the army".

With that said, in most Western stuff, they _are_ referred to as the JSDF but I also don't think it makes any material difference. Not gonna ask you to revert this.